response_writer.go

```diff
@@ -0,0 +1,97 @@
+package slacker
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/slack-go/slack"
+)
+
+// Writer interface is used to respond to an event
+type Writer interface {
```

Comment:
Why duplicate `WriterResponse` interface? As an interface name, doesn't really line up with what it does. See my comments on `WriterResponse` interface about naming.

Reply:
Got rid of the interfaces